extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -458,6 +464,28 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     }, {});
   }
 
+  nextSlide_() {
+    dev().assert(this.carousel_).getImpl().then(carousel => {
+      carousel.interactionNext();
+    });
+  }
+
+  prevSlide_() {
+    dev().assert(this.carousel_).getImpl().then(carousel => {
+      carousel.interactionPrev();
+    });
+  }
+
+  buildNavControls_() {
+    this.navControls_ = this.win.document.createElement('div');
```

Comment:
nits:

- define `controls` first and then assign it as an instance memeber
- `controls` instead of `navControls` as `nav` is inferred from context.

Reply:
The terminology here is a bit misleading. Maybe there's a better way to word this. 

Lightbox has a total of 5 buttons (open gallery, close gallery, close lightbox, next, prev). I use `controls` to refer to all of them plus the description box. Hence `controlsContainer_` contains `topBar_`, `navControls_`, and `descriptionBox_`. 

`nav` here is specified to differentiate it from `openGallery`, `closeGallery`, and `close` in `topBar_`.